readthedocs/projects/tasks/utils.py

```diff
@@ -222,10 +224,12 @@ def set_builder_scale_in_protection(builder, protected_from_scale_in, build_id=N
             AutoScalingGroupName=scaling_group,
             ProtectedFromScaleIn=protected_from_scale_in,
         )
-    except Exception:
+    except (ValidationError, ClientError):
         # Don't log these as exceptions,
         # since there isn't much we can do about it here.
-        log.info("Failed when trying to set instance protection.")
+        pass
```

Comment:
We can keep the WARNING log line here, so we know that something happened in New Relic but doesn't bother us.